Updated jenkins PR (#11)
abhayPS11/arm-learning-paths: content/learning-paths/servers-and-cloud-computing/jenkins-lp/azure_firewall.md

---
title: Create a Firewall Rule on Azure
weight: 4

### FIXED, DO NOT MODIFY
layout: learningpathall
---

## Overview

> In this section, you create a Firewall Rule within the Microsoft Azure Console to expose TCP port 8080.

Reviewer: ```suggestion
In this section, you will create a firewall rule in the Microsoft Azure Console to allow inbound TCP traffic on port 8080.
```
